ci/build.py

```diff
@@ -48,7 +49,7 @@ def get_platforms(path: Optional[str]="docker"):
     return platforms
 
 
-def get_docker_tag(platform: str) -> None:
+def get_docker_tag(platform: str) -> str:
```

Comment:
why does this script use python3 only features?

Reply:
it's run by python3 only

Reply:
mxnet promises 2 & 3 support

Reply:
This is only used in the CI environment or when a developer tries to replicate a test failure locally, means this is basically a tool. I think we're fine with separating (and deprecating) tools and the used versions from the requirements we set for MXNet itself. No user will be using this script.